lang/src/org/partiql/lang/planner/transforms/LogicalToLogicalResolvedVisitorTransform.kt

```diff
@@ -244,6 +244,26 @@ internal data class LogicalToLogicalResolvedVisitorTransform(
         }
     }
 
+    // TODO: Remove from experimental once https://github.com/partiql/partiql-docs/issues/31 is resolved and a RFC is approved
+    override fun transformBexprWindow_windowSpecification(node: PartiqlLogical.Bexpr.Window): PartiqlLogicalResolved.Over {
+        val bindings = getOutputScope(node)
```

Comment:
```suggestion
        val bindings = getOutputScope(node).concatenate(this.inputScope)
```
We *may* need to concatenate `this.inputScope`. Also, this is applicable to both added transforms.

Is the following possible?
```sql
SELECT
  (SELECT LAG(t.a) OVER(PARTITION BY t.b) FROM <<0>>)
FROM t
```

Reply:
Spot on. Fixed!